### PR TITLE
add cafe

### PR DESCRIPTION
**Pull request by linzhi2013**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by linzhi2013** [comment]
try again

**Comment by linzhi2013** [comment]
@bioconda/core

This my first time of pull request. I've checked related pages on bioconda, but do not what to do now.

Thanks

**Comment by linzhi2013** [comment]
> Repeating comment from @linzhi2013 to enable @mention:
> 
> > @bioconda/core
> > This my first time of pull request. I've checked related pages on bioconda, but do not what to do now.
> > Thanks

Sorry, but I don't get it.

**Comment by dpryan79** [comment]
> > Repeating comment from @linzhi2013 to enable @mention:
> 
> Sorry, but I don't get it.

Since you're not (yet) part of the bioconda project, you can't ping bioconda/core. The biocondabot has been programmed to look for that and ping us on your behalf in such situations (otherwise we're never notified).

**Comment by linzhi2013** [comment]
> Anyway, I've made some changes to help get this working, though I think you need to run ./configure

Hi  Devon Ryan, 

Thank you very much for your help!

@bioconda/core, thank you, too! I have been wanting to join Bioconda since long time ago, for I always want to add some missing packages to Bioconda, but I found it was a bit difficult for me to get started.

Could I possibly join Bioconda now? And I will learn more about how to write the recipes. 

Thank you!

File: recipes/cafe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - zlib
+
+test:
+  commands:
+    - which cafe
+
+about:
+  home: https://hahnlab.github.io/CAFE/
+  license: IU OPEN SOURCE LICENSE (see https://github.com/hahnlab/CAFE/blob/master/LICENSE)
+  license_file: LICENSE
```
Comment (bgruening) [inline recipes/cafe/meta.yaml]: This reference the file in the tarball ... https://github.com/hahnlab/CAFE/blob/v4.2.1/LICENSE

So I assume you can remove the copy of license file in this PR.

File: recipes/cafe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+
+test:
+  commands:
+    - which cafe
```
Comment (bgruening) [inline recipes/cafe/meta.yaml]: can you add a test like `cafe --help` or --version?

File: recipes/cafe/meta.yaml (hunk 1 above)
Comment (linzhi2013) [reply recipes/cafe/meta.yaml]: Thanks for the advice, I've remove the copy of license file in this PR.

File: recipes/cafe/meta.yaml (hunk 2 above)
Comment (linzhi2013) [reply recipes/cafe/meta.yaml]: sure, a test on the version has been added.

**Comment by dpryan79** [comment]
Thanks @linzhi2013, when this finishes testing I'll merge it. You'll receive your bioconda invite via github shortly.

**Comment by linzhi2013** [comment]
Thank you all!

**Pull request merged**